cmd/ebrelayer/relayer/cosmos.go

```diff
@@ -309,11 +314,17 @@ func getOracleClaimType(eventType string) types.Event {
 
 // Parses event data from the msg, event, builds a new ProphecyClaim, and relays it to Ethereum
 func (sub CosmosSub) handleBurnLockMsg(cosmosMsg types.CosmosMsg, claimType types.Event) {
-	sub.Logger.Info(cosmosMsg.String())
+	sub.SugaredLogger.Infow("handle burn lock message.",
+		"cosmos message is", cosmosMsg.String())
```

Comment:
json is better if the field names ("cosmos message is") don't have spaces

Reply:
accepted.